sts_endpoint_sg_rules.tf

```diff
@@ -0,0 +1,27 @@
+# Allow ingress via HTTPS from the desktop gateway security group
+resource "aws_security_group_rule" "ingress_from_desktop_gw_to_sts_via_https" {
+  provider = aws.provisionassessment
+
+  security_group_id        = aws_security_group.sts.id
+  type                     = "ingress"
+  protocol                 = "tcp"
+  source_security_group_id = aws_security_group.desktop_gateway.id
+  from_port                = 443
+  to_port                  = 443
+}
+
+# Allow ingress via HTTPS from the operations security group
+#
+# TODO - Only the Nessus instances require this access.  We could
+# avoid giving it to the other instance types by creating a separate
+# security group for Nessus instances.  See #95 for more details.
```

Comment:
Do we want to include the link or just the number? It seems like we usually include a full link.

```suggestion
# security group for Nessus instances.  See #95 for more details:
# https://github.com/cisagov/cool-assessment-terraform/issues/95
```